Tests/GRPCCodeGenTests/Internal/Renderer/TextBasedRendererTests.swift

```diff
@@ -41,7 +41,7 @@ final class Test_TextBasedRenderer: XCTestCase {
         Also, bar
         """#
       ),
-      renderedBy: TextBasedRenderer.renderComment,
```

Comment:
Curious: what warning were we getting here?

Reply:
Implicit capture of self which requires `TextBasedRendered` to be `Sendable` (which it isn't)